src/Frame.js

```diff
@@ -0,0 +1,49 @@
+function Frame(finalFrame) {
+this._frameRoll1Score = 0;
+this._frameRoll2Score = 0;
+this._frameRoll3Score = 0;
+this._bonusScore = 0;
+this._strike = false;
+this._done = false;
+this._spare = false;
+this._finalFrame = finalFrame;
+this._rollNumber = 0;
```

Comment:
You are not closing the constructor here, but instead defining all methods inside the constructor. This is not how the constructor/prototype pattern is supposed to work, so I'm surprised it actually does work... 🤔